src/content/reference/react/useSyncExternalStore.md

```diff
@@ -221,11 +222,11 @@ function subscribe(callback) {
 
 ---
 
-### Extracting the logic to a custom Hook {/*extracting-the-logic-to-a-custom-hook*/}
+### Mengekstrak logika ke *Hook* buatan sendiri {/*extracting-the-logic-to-a-custom-hook*/}
```

Comment:
```suggestion
### Mengekstrak logika ke Hook buatan sendiri {/*extracting-the-logic-to-a-custom-hook*/}
```

For loan words, don't italicise it on headings. [(reference)](https://github.com/reactjs/id.react.dev/wiki/Panduan-Penulisan-Universal#panduan-penulisan-spesifik-bahasa-indonesia)